Lib/uuid.py

```diff
@@ -740,7 +775,9 @@ def uuid8(a=None, b=None, c=None):
     int_uuid_8 = (a & 0xffff_ffff_ffff) << 80
     int_uuid_8 |= (b & 0xfff) << 64
     int_uuid_8 |= c & 0x3fff_ffff_ffff_ffff
-    return UUID(int=int_uuid_8, version=8)
+    # by construction, the variant and version bits are already cleared
+    int_uuid_8 |= _RFC_4122_VERSION_8_FLAGS
```

Comment:
```suggestion
    int_uuid_8 += _RFC_4122_VERSION_8_FLAGS
```
I think we should not make this change (the `|=` is more readable), but still fun to wonder: would it be faster due to the adaptive interpreter?

Reply:
Well.. in this case it works well but if we were to change the construction, we'll also need to update the operation. I think it's clearer with a `|` rather than a `+` although I said that the bits were cleared.

Reply:
It makes a tiny difference, but lets indeed stick with `|`
```
In [9]: %timeit int_uuid_8 | _RFC_4122_VERSION_8_FLAGS
   ...: %timeit int_uuid_8 + _RFC_4122_VERSION_8_FLAGS
43.9 ns ± 0.231 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
39.3 ns ± 0.235 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```